free-programming-books-ro.md

```diff
@@ -21,7 +21,7 @@
 
 ### HMTL
 
-* [HTML](http://tutorialehtml.com/htmlt)
+* [HTML](http://tutorialehtml.com/en/html-tutorial-complete-html-guide/)
```

Comment:
Done in https://github.com/davisonio/awesome-free-programming-books/commit/226a49895d5f897b84023e684ca2d34c3894e970